test/deprecated/typeQueryFuncs.chpl

```diff
@@ -39,3 +39,27 @@ writeln("isFloat(1.2) returns ", isFloat(1.2));
 writeln("isFloat(real) returns ", isFloat(real));
 writeln("isFloatType(real) returns ", isFloatType(real));
 writeln("isFloatValue(1.2) returns ", isFloatValue(1.2));
+
+// Type subtype relations (operators now marked unstable, will likely be
```

Comment:
Looks like this comment needs to be updated to say "deprecated" instead of "unstable"